pigar/__main__.py

```diff
@@ -197,16 +203,17 @@ def extract_reqs(self):
         logger.info('Finish local environment checking.')
         return reqs, try_imports, guess
 
-    def _write_reqs(self, reqs):
+    def _write_reqs(self, reqs, minimal=False):
         print(Color.BLUE('Writing requirements to "{0}"'.format(
             self._save_path)))
         with open(self._save_path, 'w+') as f:
             f.write('# Requirements automatically generated by pigar.\n'
                     '# https://github.com/damnever/pigar\n')
             for k, v in reqs.sorted_items():
                 f.write('\n')
```

Comment:
The empty line is for comments to improve readability, I do not think leaving it here is a good idea.

Reply:
@damnever Good point. I added a comment to fill this line.

Reply:
What is the content?

Reply:
What is the content?

Reply:
@damnever the content is just """Write requirements to a file"""

Reply:
em.. the empty line is `f.write('\n')`.

Reply:
Oh, now I see. I will not print the empty line in the requirements.txt if the user has chosen not to show referenced files.